db/Dockerfile

```diff
@@ -1,13 +1,11 @@
 FROM postgres:12.2
 
-# use uid 1000 to (hopefully) match host user (ie, you)
-RUN useradd --uid 1000 --user-group --create-home --home-dir /home/db db
-
-RUN chown db:db /var/lib/postgresql/data && \
-    chown db:db /var/run/postgresql
-
-USER db:db
+USER root:root
```

Comment:
we don't use sudo or root

Reply:
I will rethink this.

Reply:
Since /etc/passwd is owned and only writable by root, I don't see any way to do the `useradd` after the container starts and w/o also running as root.

I have another approach that goes back to baking the 'db' UID into the image, thought it seems that hasn't been an issue.  Unfortunately it involves some more conscious upfront effort.